its/ruling/src/integrationTest/java/org/sonarsource/iac/IacRulingTest.java

```diff
@@ -137,6 +137,14 @@ void testArm() throws IOException {
     runRulingTest("azureresourcemanager", properties);
   }
 
+  @Test
+  void testSpringConfig() throws IOException {
+    var properties = Map.of(
+      "sonar.inclusions", "sources/spring-config/**/*.properties,sources/spring-config/**/*.yml,sources/spring-config/**/*.yaml",
```

Comment:
I thought about narrowing it to `application*` files to avoid the noise from other issues discovered in YAMLs, however I believe we can also test how the spring-config sensor selects Spring-related files if we don't artificially filter others out.